src/gui/static/src/assets/i18n/en.json

```
@@ -370,6 +370,7 @@
     "restore-seed" : {
       "text": "Before proceeding, please select the number of words that the seed you want to recover has.",
       "check-text": "You can use this option to enter a seed and check if it is equal to the one in the hardware wallet. Before start, select the number of words the seed you want to check has.",
+      "warning" : "WARNING: to avoid potential problems, use only seeds created with a hardware wallet form the same brand/model.",
```

```suggestion
      "warning" : "WARNING: to avoid potential problems, use only seeds created with a hardware wallet from the same brand/model.",
```
but I would probably delete the "from the same brand/model" part altogether

---

I fixed it as suggested. I would like to mantain the last part because the integration with Ledger will not be compatible with our hardware wallet (at least initially). @gz-c , what do you think about this text?